react-components/src/architecture/concrete/observationsDomainObject/ObservationsView.ts

```diff
@@ -7,32 +7,47 @@ import { GroupThreeView } from '../../base/views/GroupThreeView';
 import { type CustomObjectIntersectInput, type CustomObjectIntersection } from '@cognite/reveal';
 import { type DomainObjectIntersection } from '../../base/domainObjectsHelpers/DomainObjectIntersection';
 import { Changes } from '../../base/domainObjectsHelpers/Changes';
-import { DEFAULT_OVERLAY_COLOR, SELECTED_OVERLAY_COLOR } from './constants';
 import { type DomainObjectChange } from '../../base/domainObjectsHelpers/DomainObjectChange';
+import { isDefined } from '../../../utilities/isDefined';
+import { first, sortBy } from 'lodash';
+import { type ObservationOverlay } from './types';
+import { getColorFromStatus } from './ObservationStatus';
 
 export class ObservationsView extends GroupThreeView<ObservationsDomainObject> {
   protected override calculateBoundingBox(): Box3 {
-    return this.domainObject.overlayCollection
-      .getOverlays()
+    return this.domainObject.overlayCollections
+      .flatMap<ObservationOverlay>((collection) => collection.getOverlays())
       .reduce((box, overlay) => box.expandByPoint(overlay.getPosition()), new Box3());
   }
 
   protected override addChildren(): void {
-    this.addChild(this.domainObject.overlayCollection);
+    this.domainObject.overlayCollections.forEach((collection) => {
+      this.addChild(collection);
+    });
   }
 
   public override update(change: DomainObjectChange): void {
     super.update(change);
```

Comment:
Listen to `Changes.clipping` here. When the point is outside the clipping planes set it invisible. Remember to invalidate the bounding box by `this.invalidateBoundingBox();`  Use `renderTarget?.getGlobalClippingPlanes()` to get the planes.